sunpy/instr/goes.py

```diff
@@ -200,18 +200,17 @@ def calculate_temperature_em(goests, abundances="coronal",
     using the methods of White et al. (2005) who used the
     CHIANTI atomic physics database to model the response of the ratio
     of the short (0.5-4 angstrom) to long (1-8 angstrom) channels of the
-    XRSs onboard various GOES satellites.  This method assumes an
-    isothermal plasma, the ionisation equilibria of
+    XRSs onboard various GOES satellites. This method assumes an
+    isothermal plasma, the ionization equilibria of
     [2]_, and a constant density of 10**10 cm**-3.
     (See [1]_ for justification of this last assumption.)
-    This function is based on goes_chianti_tem.pro in SolarSoftWare
+    This function is based on "goes_chianti_tem.pro" in SolarSoft
```

Comment:
"SolarSoftWare" is the full name. Do we consistently use "SolarSoft" elsewhere in the package?